### PR TITLE
feat(splashscreen): Add support for spinner on Android and iOS

### PR DESCRIPTION
**Pull request by trancee**
This pull request will add support for displaying a spinner on top of the splash screen, both for Android and iOS.

There are some new configuration keys:
- showSpinner *bool*: if `true` will enable the spinner, defaults to `false`.
- spinnerStyle *string*: controls the style of the spinner. 
Android: `horizontal`, `small`, `large`, `inverse`, `smallInverse`, `largeInverse`. 
iOS < 13: `whiteLarge`, `large`, `gray`. 
iOS >= 13: `large`, `medium`.
- spinnerColor *string*: sets the color of the spinner, values are either `#RGB` or `#ARGB`.

Refers also to issue #928.

### CODE REVIEW

File: ios/Capacitor/Capacitor/Plugins/SplashScreen.swift [hunk 1]
```diff
@@ -1,149 +1,270 @@
 import Foundation
 import AudioToolbox
 
+// https://stackoverflow.com/questions/24263007/how-to-use-hex-color-values
+extension UIColor {
```
[the remark this answers is not in the document]
Comment (trancee) [reply ios/Capacitor/Capacitor/Plugins/SplashScreen.swift]: Thank you for your feedback, @jcesarmobile.

Please have a look at the changes, I hope I made it all right this time. Let me know if you have any other suggestions.

**Comment by trancee** [comment]
@jcesarmobile I am using `swift-format` as suggested in this blog post:
https://nshipster.com/swift-format/#swift-format

This is the only one that uses 2 spaces instead of 4. Even if do copy/paste of code in Xcode itself, it will change to 4 spaces. I am not sure what kind of code formatter you are using, but I am open for any suggestion to do it the proper way.

**Comment by jcesarmobile** [comment]
I don't use any formatter.
You can configure Xcode to use 2 spaces instead of 4 globally, or set it per file before editing it (probably this is better for you as you probably use 4 in other places)
See screenshot showing both the global preference and the per file on the right side
Captura de pantalla 2019-06-13 a las 15 10 24

**Comment by trancee** [comment]
@jcesarmobile Thank you for the suggestion, it works that way in Xcode with 2 spaces indentation.

**Comment by jcesarmobile** [comment]
I've made a few code changes on the iOS code fixing some mistakes and code formatting.

Also simplified the iOS styles to just large and small, making large the default to match android default, because as they can be colored I think it doesn't make sense to have a grey option that you can change to another color. 

Android code looks good, going to merge, thanks!

**Pull request merged**